conda_build/post.py

```diff
@@ -1732,6 +1732,54 @@ def fix_permissions(files, prefix):
                 log.warn(str(e))
 
 
+def check_menuinst_json(files, prefix):
```

Comment:
I'm a little confused about the purpose of this function. It looks like it attempts to validate the menuinst JSON file, but it doesn't return anything. I would expect this type of function to return a boolean value or perhaps an error message. Instead it appears to just be using the logger to print this information.

This is different enough from what I would expect that it warrants a brief description as a doc string. Also consider annotating it with a return type to make that a little less ambiguous.

Reply:
Sounds good. I know conda-build is a weird. It's appropriate to have different standards for this project.